accelerator/core/proxy_apis.c

```diff
@@ -302,6 +302,7 @@ status_t proxy_api_wrapper(const ta_config_t* const iconf, const iota_client_ser
                            const char* const obj, char** json_result) {
   status_t ret = SC_OK;
   if (iconf->proxy_passthrough) {
+    ta_log_debug("Proxy_passthrough enabled.\n");
```

Comment:
I am ok with both, but the API `info` should provide all these messages which is what we haven't implemented yet

Reply:
I think we could show this message only once when TA starting up.
This log here will show the message every time we get a new request.
Btw, we might change `proxy_passthrough` as the default mode.

Reply:
My original concern is that we could print the configuration setting on startup only once.
When it comes to debugging, this message could remind programmer that this request is handled in this `proxy_passthrough`-enabled block or not.

In production, we could simply ignore the message, so I use `debug` instead of `info` here.

Reply:
So how about removing this one?